src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier firstFigure = new FigureSupplier();
+        FigureSupplier secondFigure = new FigureSupplier();
+        FigureSupplier thirdFigure = new FigureSupplier();
+        firstFigure.getRandomFigure();
+        secondFigure.getRandomFigure();
+        thirdFigure.getRandomFigure();
```

Comment:
use a loop to create random Figure, for this you only need one variable of type FigureSupplier